Tasks/Common/VstsAzureHelpers_/Tests/Initialize-AzureSubscription.ManagedServiceIdentity.ps1

```diff
@@ -48,9 +52,17 @@ foreach ($variableSet in $variableSets) {
     Register-Mock Set-UserAgent
     Register-Mock Invoke-WebRequest { $response }
     
-    # Act.
-    $result = & $module Initialize-AzureSubscription -Endpoint $endpoint -StorageAccount $variableSet.StorageAccount
+    if ($featureFlags.retireAzureRM) {
+        try {
+            & $module Initialize-AzureSubscription -Endpoint $endpoint -StorageAccount $variableSet.StorageAccount
```

Comment:
What if this doesn't throw? Shouldn't we add some always failing assert after this?

Reply:
Added throw in case if Initialize-AzureSubscription will finish without errors, thanks!